packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -379,17 +378,21 @@ export default class MultiSelect extends React.Component {
                             key={itemProps.id}
                             isActive={isChecked}
                             isHighlighted={highlightedIndex === index}
+                            role="group"
+                            aria-label={itemProps.id}
                             {...itemProps}>
-                            <Checkbox
-                              id={`${itemProps.id}__checkbox`}
-                              title={useTitleInItem ? itemText : null}
-                              name={itemText}
-                              checked={isChecked}
-                              disabled={disabled}
-                              readOnly={true}
-                              tabIndex="-1"
-                              labelText={itemText}
-                            />
+                            <div
+                              role="group"
+                              aria-label={`${itemProps.id}__checkbox`}
```

Comment:
We should be able to remove these groups, right? And then the item itself should be getting `aria-selected` from Downshift, right?

Reply:
I can try removing it and see if DAP complains

Reply:
removing it doesn't cause any DAP errors in my testing here

Reply:
Seems like `aria-selected` was not being passed down via downshift. If I add it, it complains that `aria-selected` can't be used on a `div`. Using `aria-checked` works though, so I've added that. About to push up some changes and we can re-test

Reply:
I believe `aria-selected` needs to be on the node with `role="option"` IIRC. Let me know if not 🤔